src/app/util/af-types.h

```diff
@@ -24,6 +24,8 @@
  */
 
 #include "att-storage.h"
+#include <platform/CHIPDeviceConfig.h> // For CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT
```

Comment:
please include an extra dependency on `src/platform:platform_buildconfig` in https://github.com/project-chip/connectedhomeip/blob/master/src/app/util/BUILD.gn#L58 (so that the dependency is direct rather than indirect)

Reply:
tried, but failed:

```
ERROR at //src/app/util/BUILD.gn:56:1: Dependency not allowed.
source_set("af-types") {
^-----------------------
The item //src/app/util:af-types
can not depend on //src/platform:platform_buildconfig
because it is not in //src/platform:platform_buildconfig's visibility list: [
  //src/platform:platform_config_header
  //src/ble:ble_config_header
  //src/system:system_config_header
]
```

I don't see enough through the gn complexity to see what would be needed to make it work

Reply:
https://github.com/project-chip/connectedhomeip/blob/master/src/platform/BUILD.gn#L431

It seems you need to use `platform_config_header`